definitions/checks/check_epel_repository.rb

```diff
@@ -0,0 +1,18 @@
+class Checks::CheckEpelRepository < ForemanMaintain::Check
+  metadata do
+    label :check_epel_repository
+    description 'Check if EPEL repoistory enabled on system'
+    tags :default
+  end
+
+  def run
+    epel_enabled?
```

Comment:
Nitpick: I would expect the `epel_enabled?` only to return `true`/`false`, and raising the exception based on the result here.